### DISCUSSION TITLE
A stupid Q: how to hide/disable the echo of input?

### DISCUSSION THREAD

**Discussion by x1a0** [Blightmud/Blightmud#589] [Q&A] [answered]
How to hide/disable the echo of input?

**Comment by LiquidityC** [comment]
You setup an input listener and gag the lines there. `/help mud`.

I can paste a code example when I'm at my computer.

**Comment by LiquidityC** [comment]
Here's a scripted solution. Not tested but I think it should work.

```lua
mud.add_input_listener(function (line)
    line:gag(true)
    return line
end)
```

**Comment by x1a0** [comment]
Thanks - works greatly!
(should be `line:gag(true)`)

**Comment by LiquidityC** [comment, marked as answer]
There is now a setting for this `/set echo_input off`
